packages/eslint-config-airbnb-base/rules/imports.js

```diff
@@ -13,11 +13,12 @@ module.exports = {
   settings: {
     'import/resolver': {
       node: {
-        extensions: ['.js', '.json']
+        extensions: ['.js', '.mjs', '.json']
       }
     },
     'import/extensions': [
       '.js',
+      '.mjs',
```

Comment:
I checked to be sure; order [doesn't matter here](https://github.com/benmosher/eslint-plugin-import/blob/dc3609f895f5ad390d88fd9925dce40960117930/utils/ignore.js#L22)